src/Liquidator.sol

```diff
@@ -99,6 +101,20 @@ contract Liquidator is Owned, ILiquidator {
         emit AuctionCurveParametersSet(999_807_477_651_317_446, 14_400, 15_000, 6000);
     }
 
+    /*///////////////////////////////////////////////////////////////
+                    AUCTION ASSET RECIPIENT
+    ///////////////////////////////////////////////////////////////*/
+
+    /**
+     * @notice The asset recipient receives all assets of an Account after an unsuccessful auction.
+     * @param assetRecipient_ The address of the new asset recipient.
+     * @dev This function can only be called by the protocol owner.
```

Comment:
Comment says protocol owner, code riskManager. Or just fetch it from the Creditor, is for a flow that should not or very rare happen?